spp_smile_audit/models/ir_module_module.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, models
```

Comment:
@reichie020212 @emjay0921  I don't see the using of this file since it is not imported in __init__ file. But I don't know if it should be removed or not